### ISSUE TITLE
Stack Pivot functionality not present

### ISSUE THREAD

**Issue by kevross33** [cuckoosandbox/cuckoo#1469]
Hi,

Exploit detection (stack pivot for ROP) was merged from pull request 17 https://github.com/cuckoosandbox/monitor/pull/17. I have noticed however it has never been included in main branch yet and in new version either. I am wondering if this could please be merged into main branch for general use?

**Comment by jbremer** [comment]
Yes, I noticed the same. We have a bunch of functional tests in place that I have to run past before I can push the latest version of Monitor - I hope to do this soon'ish, though :-) Thanks for reminder!
